index.bs

```diff
@@ -1962,24 +1962,55 @@ Note: While APIs should not
 expose a full list of devices in an [=implementation-defined=] order,
 they may need to for web compatibility reasons.
 
-<h3 id="wrapper-apis">Native APIs don't typically translate well to the web</h3>
+<h3 id="usecase-oriented-apis">Design based on functionality, not the underlying API or hardware</h3>
 
+Try to design new native capabilities being brought to the web based on functionality.
 
-When adapting native operating system APIs for the web,
-make sure the new web APIs are designed with web platform principles in mind.
+You will probably not be able to directly translate
+an API available to native applications
+to a web API.
+
+Instead, consider the functionality available from the native API,
+and the user needs it addresses,
+and design an API that meets those user needs,
+even if the implementation depends on the existing native API.
+
+Be particularly careful about exposing
+the exact lifecycle and data structures of the underlying native APIs.
```

Comment:
I wonder if we want to say why. That probably makes it easier to understand. Like native APIs differ, and new ones might appear in the future that works quite differently. It is quite common to just deprecated APIs in native land, but not on the web

Reply:
We should address this in a followup PR - will discuss in https://github.com/w3ctag/design-principles/issues/333.